spark/interpreter/src/main/java/org/apache/zeppelin/spark/SparkRInterpreter.java

```diff
@@ -156,7 +155,15 @@ public InterpreterResult interpret(String lines, InterpreterContext interpreterC
           "\", \"" + jobDesc + "\", TRUE)";
     }
     lines = setJobGroup + "\n" + lines;
-
+    if (sparkInterpreter.getSparkVersion().newerThanEquals(SparkVersion.SPARK_2_3_0)) {
```

Comment:
what happens when the spark version is not newer than 2.3.0? would it make the pooling not work?

Reply:
then, we should document this instead of dropping silently

Reply:
Sure, will add doc